### ISSUE TITLE
Docker compose start with selected service 

### ISSUE THREAD

**Issue by MCatherine1994** [testcontainers/testcontainers-python#315]
Hello there, I wonder if we could run the docker container based on a docker compose file, but only run the selected service inside it? like doing this command `docker-compose up [service_name]`, is there any method can support that?

Currently we're using:
```
import testcontainers.compose
compose = testcontainers.compose.DockerCompose(COMPOSE_PATH)
compose.start()
```
But it will start all the services, but we just want to start selected services.

Thanks!!

**Comment by tillahoffmann** [comment]
Yes, that should be possible in principle by modifying the `start` function. Contributions welcome!

https://github.com/testcontainers/testcontainers-python/blob/6371c83fb1d390070132e50271830b31db6ea14e/compose/testcontainers/compose/__init__.py#L101-L113.

**Comment by balint-backmaker** [comment]
@tillahoffmann I'd be interested in picking this up, just the matter of fitting it into my usual duties at the company I work for.

Having said that, a simple "services" component should be a no-brainer! Expect a PR soon! :)
